DigitalLearningSolutions.Web/Controllers/TrackingSystem/Delegates/DelegateGroupsController.cs

```diff
@@ -12,19 +19,72 @@
     [Route("TrackingSystem/Delegates/Groups")]
     public class DelegateGroupsController : Controller
     {
+        private const string DelegateGroupsFilterCookieName = "DelegateGroupsFilter";
+        private static readonly DateTimeOffset CookieExpiry = DateTimeOffset.UtcNow.AddDays(30);

```

Comment:
I wonder if it would be worth defining a standard FilterCookieExpiry somewhere and use it for all filters?